build.gradle

```diff
@@ -312,6 +312,7 @@ ext.initScreenshotTests = { project ->
     if (hasScreenshots) {
         project.apply plugin: 'app.cash.paparazzi'
     }
+    project.dependencies { testCompileOnly "app.cash.paparazzi:paparazzi:1.0.0" }
```

Comment:
the paparazzi jar is automatically applied by plugin however we only apply the plugin when running the screenshot tasks (because the jar causes runtime class clashes when running the unit tests)

but... we need to always have the jar on the compilation class path so that we can compile the unit tests source set, hence the `testCompileOnly` declaration